### DISCUSSION TITLE
Use Vscodium as difftool with chezmoi

### DISCUSSION THREAD

**Discussion by laguill** [VSCodium/vscodium#2108] [Q&A]
Hello,

I am managing my dotfiles using [chezmoi](https://www.chezmoi.io/quick-start/#set-up-a-new-machine-with-a-single-command)

It seams that when vscodium is called from external tool the file cannot be found and tried to open it from /tmp
See [#issue4085](https://github.com/twpayne/chezmoi/issues/4085) for more details

Any idea how to solve this ?

Thanks

**Comment by daiyam** [comment]
It seems you are making good progress on the chezmoi's issue. It seems you should be able to fix it soon. Good speed ;)

**Comment by laguill** [comment]
Solved by changing tempdir of chezmoi
See [#issue4085](https://github.com/twpayne/chezmoi/issues/4085)
